src/main/java/hw_6/Task1.java

```diff
@@ -0,0 +1,42 @@
+package hw_6;
+
+public class Task1 {
+    public static void main(String[] args) {
+
+        Task1 task = new Task1();
+        task.NonStaticMethod();
+
+        StaticMethod();
+
+    }
+
+    int x = 3;
+    int y = 4;
```

Comment:
Where is access modifiers?) Please add in all places

Reply:
fixed